Sample/UIKit/UserInfoViewController.swift

```diff
@@ -6,24 +6,40 @@ class UserInfoViewController: UIViewController {
     @IBOutlet private weak var userIDTextField: UITextField!
     @IBOutlet private weak var idTrackingIdentifierTextField: UITextField!
     @IBOutlet private weak var accessTokenTextField: UITextField!
+    private var userInfo: UserInfo?
```

Comment:
you can make this non optional and assign `UserInfo()` as value.
Then you can remove the condition in `saveUserInfoAction()` and call  `RInAppMessaging.registerPreference(userInfo)` in init or viewDidLoad